spyderlib/spyder.py

```diff
@@ -1144,7 +1146,11 @@ def add_xydoc(text, pathlist):
                                          self.close_dockwidget_action,
                                          self.maximize_action,
                                          None))
+            self.toggle_toolbars_action = create_action(self,
+                                    _("Toggle toolbar visibility"),

```

Comment:
Suggestion here: `Show toolbars`. I think it's easier to understand to what it refers to ;-)